packages/expo-passkeys/android/build.gradle

```
@@ -0,0 +1,53 @@
+apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android'
+apply plugin: 'maven-publish'
+
+group = 'expo.modules.clerkexpopasskeys'
+version = '0.1.0'
```

maybe we should go for:
```suggestion
version = '1.0.0'
```